quickstarts/hello-rules/README.md

```diff
@@ -1,8 +1,6 @@
 ## IBM Business Automation Manager Open Editions KIE Server decisions Quickstart
 
-Note that, this is the development branch, the target images might not be available here, instead you can look at the [released branch](https://github.com/jboss-container-images/rhpam-7-openshift-image/tree/7.13.x-blue/quickstarts/hello-rules)
-
-This quickstart is intended to be used with the [IBM BAMOE KIE Server](https://github.com/jboss-container-images/rhpam-7-openshift-image/tree/main/kieserver) image.
+This quickstart is intended to be used with the [IBM BAMOE KIE Server](https://github.com/jboss-container-images/rhpam-7-openshift-image/tree/8.0.3.GA/kieserver) image.
```

Comment:
Shouldn't the link be https://github.com/jboss-container-images/rhpam-7-openshift-image/tree/7.13.x-blue/kieserver ?